src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexLexer.cs

```diff
@@ -364,12 +364,18 @@ private bool IsOptionChar(char ch)
         {
             switch (ch)
             {
-                case '+': case '-':
-                case 'i': case 'I':
-                case 'm': case 'M':
-                case 'n': case 'N':
-                case 's': case 'S':
-                case 'x': case 'X':
+                case '+':
+                case '-':
+                case 'i':
+                case 'I':
+                case 'm':
+                case 'M':
+                case 'n':
+                case 'N':
+                case 's':
+                case 'S':
+                case 'x':
+                case 'X':
```

Comment:
:'(

Note that the original formatting was intentional.  Though i can live with this.

Reply:
Do we not have: 

![image](https://user-images.githubusercontent.com/4564579/48507005-99559680-e7ff-11e8-9151-25b11917c966.png)

On?  I would consider this a version of that rule.

Reply:
Note: i'm still ok with this.  I prefer the previous code.  But i wouldn't block the formatting analyzer on this.